test/support/test_helpers.ex

```diff
@@ -1,9 +1,25 @@
 defmodule CodeCorps.TestHelpers do
   use Phoenix.ConnTest
+  import ExUnit.Assertions
 
   def ids_from_response(response) do
     Enum.map response["data"], fn(attributes) ->
       String.to_integer(attributes["id"])
     end
   end
+
+  def assert_jsonapi_relationship(json = %{"relationships" => relationships}, relationship_name, id) do
+    assert relationships[relationship_name]["data"]["id"] == Integer.to_string(id)
+    json
+  end
+
+  def assert_jsonapi_relationship(json, relationship_name, id) do
```

Comment:
Added 2 of these methods. One to be used when passing the relationships itself (above) and one where we can pass the whole response.